db/index.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
What is this code for?

Reply:
it restarts the db when I want to restart it. I'd want to restart it if i make changes to models